ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -1010,6 +1012,10 @@ export default defineComponent({
   font-size: 20px !important;
 }
 
+.pink-font{
```

Comment:
We don't have pink font in the app. The figma design has pink font to highlight what is changing.

Reply:
Haha, my bad. Fixed!